content/hardware/04.pro/boards/portenta-h7-lite-connected/downloads/ABX00042-full-pinout.pdf

```diff
@@ -0,0 +1 @@
+../../../../04.pro/boards/portenta--h7/downloads/ABX00042-full-pinout.pdf
```

Comment:
These are not shown in the preview. I'm not sure we can symlink downloadables like this, are we doing it for another product where I can check?

Reply:
to check in the future, it works for tutorials but I guess the backed is not prepared for "downloadables"

good point indeed , thanks